File: openbis_api/source/java/ch/ethz/sis/openbis/generic/as/api/v3/dto/common/update/IdListUpdateValue.java
package ch.ethz.sis.openbis.generic.as.api.v3.dto.common.update;

import ch.systemsx.cisd.base.annotation.JsonObject;

/**
 * @author pkupczyk
 */
@JsonObject("dto.common.update.IdListUpdateValue")
public class IdListUpdateValue<T> extends ListUpdateValue<T, T, T, T>
{
    private static final long serialVersionUID = 1L;

}

File: plasmid/source/core-plugins/eln-lims/1/as/webapps/eln-lims/html/lib/openbis/js/v3/as/dto/common/update/IdListUpdateValue.js
/**
 * @author pkupczyk
 */
define([ "stjs", "as/dto/common/update/ListUpdateValue" ], function(stjs, ListUpdateValue) {
	var IdListUpdateValue = function() {
		ListUpdateValue.call(this);
	};
	stjs.extend(IdListUpdateValue, ListUpdateValue, [ ListUpdateValue ], function(constructor, prototype) {
		prototype['@type'] = 'as.dto.common.update.IdListUpdateValue';
		constructor.serialVersionUID = 1;
	}, {
		actions : {
			name : "List",
			arguments : [ {
				name : "ListUpdateValue.ListUpdateAction",
				arguments : [ "ACTION" ]
			} ]
		}
	});
	return IdListUpdateValue;
})